adafruit_as7341.py

```diff
@@ -315,7 +331,7 @@ def initialize(self):
         self.gain = Gain.GAIN_128X  # pylint:disable=no-member
 
     @property
-    def all_channels(self):
+    def all_channels(self) -> Struct:
```

Comment:
Is there a reason here not to just do `Tuple[int, ...]`? That seems safer. Thoughts?

Reply:
Hmm, good question!  Looking into it, as long as it's a homogeneous tuple you can!  It's definitely the way to go for expressing arbitrary length tuples.  I'd say as long as the documentation is clear that it will return a tuple of the channels, it does seem cleaner!

Reply:
Perfect, we'll go with that then. Thanks!